thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/dashboard/views/tabs/analysis/rootcause-table.ftl

```diff
@@ -1,4 +1,22 @@
 <div class="analysis-card padding-all top-buffer">
   <h4 class="analysis-title bottom-buffer">[BETA] Root Cause Search</h4>
-  <table id="rootcause-data-table" class="table table-borderless rootcause__table"></table>
+  <ul class="nav nav-tabs" role="tablist">
+    <li><a class="active" data-toggle="tab" data-selector="">All</a></li>
```

Comment:
can we have classnames for each `a` so that the js is a little bit more specific than `a[data-toggle="tab"]` 
for example `.rca-link[data-toggle="tab"]`